app/subscriber/src/features/commentary/Commentary.tsx

```diff
@@ -112,19 +111,7 @@ export const Commentary: React.FC = () => {
           </Row>
         }
       >
-        <div className="content">
-          {commentary.map((x) => {
-            return (
-              <Row key={x.id} className="content-row">
-                <Sentiment value={x.tonePools?.length ? x.tonePools[0].value : 0} />
-                <DetermineContentIcon contentType={x.contentType} />
-                <div className="headline" onClick={() => navigateAndScroll(`/view/${x.id}`)}>
-                  {x.headline}
-                </div>
-              </Row>
-            );
-          })}
-        </div>
+        <ContentList selected={[]} onContentSelected={noop} content={commentary} simpleView />
```

Comment:
I tried making selected, and onContentSelected optional props; however that resulted in many errors. For now passing an empty array and noop. Will create a ticket to make this more flexible.